### PR TITLE
Add Wagtail Space NL 2022 banner to docs

### PR DESCRIPTION
**Pull request by allcaps**
Update of text and date to match the next upcoming event.
I also made the cookie name unique and moved it to a variable.

https://www.wagtail.space/2022/

### CODE REVIEW

File: docs/_static/js/banner.js [hunk 1]
```diff
@@ -16,23 +16,24 @@ function hasCookie(name, value) {
 $(() => {
   const $wagtailspace = $(`
   <div class="wagtailspace">
-    <a href="https://us.wagtail.space/">
+    <a href="https://wagtail.space/2022/">
```
Comment (Scotchester) [inline docs/_static/js/banner.js:19]: This is extremely nitpicky, but adding the `www.` here would avoid a redirect :)

```suggestion
    <a href="https://www.wagtail.space/2022/">
```

File: docs/_static/js/banner.js [hunk 2]
```diff
@@ -16,23 +16,24 @@ function hasCookie(name, value) {
...
       <div>
-        <strong>Wagtail Space US</strong><br> 24 – 26 March 2022 online and in Cleveland, Ohio
+        <strong>Wagtail Space NL</strong><br> 15 – 17 June 2022 in Arnhem, The Netherlands
```
[the remark this answers is not in the document]
Comment (allcaps) [reply docs/_static/js/banner.js:22]: Not promoting it (yet).

**Comment by caspertimmers** [comment]
👍

**Review by caspertimmers — APPROVED** [review]

**Review by thibaudcolas — APPROVED** [review]
yay!

**Pull request merged**